examples/react/project.json

```diff
@@ -11,7 +11,7 @@
         "compiler": "babel",
         "outputPath": "dist/examples/react",
         "index": "examples/react/src/index.html",
-        "baseHref": "/",
+        "baseHref": "/wallet-selector/",
```

Comment:
I suspect this would change the React example locally too? I don't really a have solution to avoid this, but maybe we should apply this to the Angular example too for consistency

Reply:
Yes it changes the React example locally too.